x-pack/test/functional/apps/upgrade_assistant/overview_page.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function upgradeAssistantOverviewPageFunctionalTests({
+  getService,
+  getPageObjects,
+}: FtrProviderContext) {
+  const PageObjects = getPageObjects(['upgradeAssistant', 'common']);
+  const retry = getService('retry');
+  const security = getService('security');
+  const testSubjects = getService('testSubjects');
+  const es = getService('es');
+
+  describe('Overview Page', function () {
+    this.tags('skipFirefox');
+
+    before(async () => {
+      await security.testUser.setRoles(['superuser', 'global_upgrade_assistant_role']);
+    });
+
+    after(async () => {
+      await security.testUser.restoreDefaults();
+    });
+
+    beforeEach(async () => {
+      await PageObjects.upgradeAssistant.navigateToPage();
+
+      await retry.waitFor('Upgrade Assistant overview page to be visible', async () => {
+        return testSubjects.exists('overview');
+      });
+    });
+
+    it('Should render all steps', async () => {
+      testSubjects.exists('backupStep-incomplete');
+      testSubjects.exists('fixIssuesStep-incomplete');
+      testSubjects.exists('fixLogsStep-incomplete');
+      testSubjects.exists('upgradeStep');
+    });
+
+    describe('fixLogsStep', () => {
+      before(async () => {
+        // Access to system indices will be deprecated and should generate a deprecation log
+        await es.transport.request({
+          method: 'GET',
+          path: '/.apm-agent-configuration',
```

Comment:
Are we confident this system index will always exist when we run the tests?

Reply:
Good one! I've changed it to `.kibana` which I believe will be around for a while